app/src/main/kotlin/at/bitfire/davdroid/resource/LocalJtxCollectionStore.kt

```diff
@@ -86,4 +88,19 @@ class LocalJtxCollectionStore @Inject constructor(
         localCollection.delete()
     }
 
+    override fun updateAccount(
+        provider: ContentProviderClient,
+        oldAccount: Account,
+        newAccount: Account
+    ) {
+        val values = contentValuesOf(JtxContract.JtxCollection.ACCOUNT_NAME to newAccount.name)
+        val uri = JtxContract.JtxCollection.CONTENT_URI.asSyncAdapter(oldAccount)
+
+        provider.update(
+            uri,
+            values,
+            "${JtxContract.JtxCollection.ACCOUNT_NAME}=?", arrayOf(oldAccount.name)
+        )
```

Comment:
Maybe one line for the call, then we can omit the blank line before and everything is much shorter.